presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -250,15 +257,27 @@ public Ssl()
         }
     }
 
-    private static class SslKeyStorePath
-            extends AbstractConnectionProperty<String>
+    private static class SslVerification
+            extends AbstractConnectionProperty<SslVerificationMode>
     {
         private static final Predicate<Properties> IF_SSL_ENABLED =
                 checkedPredicate(properties -> SSL.getValue(properties).orElse(false));
 
+        static final Predicate<Properties> IF_SSL_VERIFICATION_ENABLED =
+                IF_SSL_ENABLED.and(checkedPredicate(properties -> !SSL_VERIFICATION.getValue(properties).orElse(SslVerificationMode.FULL).equals(SslVerificationMode.NONE)));
+
```

Comment:
Static import is not applied to `SslVerificationMode.FULL` and `SslVerificationMode.NONE` as `io.prestosql.client.ClientSelectedRole.Type.NONE` has already been imported.